R/awsjavasdk-internal.R

```diff
@@ -17,8 +17,9 @@
 .app_dir <- rappdirs::app_dir("awsjavasdk", version = packageVersion("awsjavasdk"))
 
 .onAttach <- function(libname, pkgname) {
+  set_sdk_file_root()
   if (!aws_sdk_present()) {
-    packageStartupMessage("awsjavasdk did not detect its own copy of the AWS SDK; you must execute install_aws_sdk() in order to download the SDK and for the package to work as expected")
+    packageStartupMessage(paste0("awsjavasdk did not detect its own copy of the AWS SDK. You must execute install_aws_sdk() in order to download the SDK and for the package to work as expected. If you've previously installed to a non-standard location, call set_sdk_file_root({{your alt_install root}}, confirm = TRUE) to point this package to that location and then run awsjavasdk::load_sdk()"))
```

Comment:
Yeah, I was planning to do exactly what you have just suggested: bundling the jar files with the package and pushing to CRAN on every new minor version (like once a year). If someone needs a newer version, then installing from GH/drat should be easy -- so I think this would be a fairly good compromise both to users, CRAN maintainers and R devs.

I'm putting together a build job that can automate this by checking https://github.com/aws/aws-sdk-java/releases for new versions, building the jars and bundling those in a new version of the R package tagged on GH with the AWS version number, so that it'd be easy to install any version of the jars from R.

Will report back here if I have anything useful.